### DISCUSSION TITLE
Help with the server sounds

### DISCUSSION THREAD

**Discussion by Dizey** [kus/cs2-modded-server#159] [Q&A]
How do I turn down the sounds on the server, in GunGame mode, when you kill 3 times, the bell starts to scream, how do I turn it down?

**Comment by kus** [comment]
I have the multi level sound (3 kills) empty, it shouldn’t make the bell sound.

Are you using your own config or the supplied one?

You can disable GunGame sounds with !music and Quake sounds with !qs

**Comment by ReneRebsdorf** [comment]
snd_toolvolume 0.25

reduces the volume of the retake announcer to 25%, I assume without verifying that it would work the same for GunGame.
Those above is a client variable, I am not aware of a server side one to set it 'globally'.
